run.sh

```diff
@@ -24,5 +24,4 @@ create)
     ;;
 esac
 bundle exec rake import:all
```

Comment:
update: have raise https://github.com/ministryofjustice/courtfinder/issues/261 about this